package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "@cloudfour/hbs-helpers": "^0.9.0",
     "@mozilla-protocol/assets": "^1.0.1",
-    "@mozilla-protocol/tokens": "^2.0.0",
+    "@mozilla-protocol/tokens": "^2.1.0",
```

Comment:
This is a change that will require refactoring, so it should probably be a major version bump. 

Still, I probably wouldn't change the version number in this PR unless you plan on publishing to NPM straight away. There may be other changes we can get into a major release?

If you do want to publish right away, you need to follow the [instructions in the docs](https://github.com/mozilla/protocol/tree/master/docs#publishing-to-npm) to do it fully, as changing this alone isn't quite enough. You'll also need to make sure you're set up as a team contributor on NPM.